autoload/leaderf/python/leaderf/fileExpl.py

```diff
@@ -28,6 +29,25 @@ def deco(*args, **kwargs):
             return func(*args, **kwargs)
     return deco
 
+def showDevIcons(func):
+    @wraps(func)
+    def deco(*args, **kwargs):
+        if lfEval("get(g:, 'Lf_ShowDevIcons', 0)") == "1":
+            content = func(*args, **kwargs)
+            if isinstance(content, AsyncExecutor.Result):
+                return content
+            else:
+                return [
+                    line if isStartDevIcons(line) else format_line(line)
+                    for line in content
+                ]
+        else:
+            return func(*args, **kwargs)
+
+    return deco
+
+def format_line(line):
+    return webDevIconsGetFileTypeSymbol(line) + line
```

Comment:
It's better to add a space after the icon, because if `set ambiwidth=double` is not set, it doesn't look beautiful.

Reply:
Changed 4f500c59623836e9f5f4595dbae098af1e84da1e.
I'm a little curious, but if I `set ambiwidth=double`, I think the icon will be halved, is that okay?

Reply:
I think the code has already handle this scenario. If the icon is halved, `set ambiwidth=double` is recommended. On my Linux system, the icon is not halved even though `set ambiwidth=double` is not set.

Reply:
So it behaves differently depending on the OS. I didn't know that.
Thank you very much.